sdk/spring/azure-spring-boot-samples/azure-spring-cloud-appconfiguration-conversion-sample-initial/README.md

```diff
@@ -86,9 +86,9 @@ In this section, you clone a containerized Spring Boot application and test it l
 1. Replace below properties in `application.properties` with information from your database.
 
    ```properties
-   azure.cosmosdb.uri=your-cosmosdb-uri
-   azure.cosmosdb.key=your-cosmosdb-key
-   azure.cosmosdb.database=your-cosmosdb-databasename
+   azure.cosmos.uri=your-cosmos-uri
```

Comment:
So this will bring breaking changes.

Reply:
should change back to `azure.cosmosdb.xxx`?